src/components/calcite-button/calcite-button.tsx

```diff
@@ -150,12 +134,19 @@ export class CalciteButton {
 
     return (
       <Tag
-        {...attributes}
+        aria-label={this.el.getAttribute("aria-label")}
         class={{ [CSS_UTILITY.rtl]: dir === "rtl", [CSS.contentSlotted]: this.hasContent }}
         disabled={this.disabled}
+        href={this.childElType === "a" && this.href}
+        name={this.childElType === "button" && this.el.getAttribute("name")}
         onClick={this.handleClick}
         ref={(el) => (this.childEl = el)}
+        rel={this.childElType === "a" && this.el.getAttribute("rel")}
+        role={this.el.getAttribute("role")}
```

Comment:
We should set the role for the user as a convenience I think.

Reply:
What should it be?

Reply:
Is it even needed? A button shouldn't need a role of button.